frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -80,6 +83,7 @@ export const cleanFunnelParams = (filters: Partial<FilterType>, discardFiltersNo
             : {}),
         ...(filters.funnel_window_interval ? { funnel_window_interval: filters.funnel_window_interval } : {}),
         ...(filters.funnel_order_type ? { funnel_order_type: filters.funnel_order_type } : {}),
+        ...(filters.hidden_map ? { hidden_map: filters.hidden_map } : {}),
```

Comment:
Nit that won't hold up the PR: could we find a name that better describes what this does? The `map` here doesn't really say what is hidden. Even `hiddenLines` would be more descriptive, if not slightly misleading...

Reply:
Hm, would `hiddenLegendKeys` work in this context?